wrappers/python/nntile/model/gpt2_block.py

```diff
@@ -0,0 +1,146 @@
+        activations = [x] + input_norm.activations_output + \
+                      attention_layer.activations_output + \
+                      post_attn_add.activations_output + \
+                      post_attn_norm.activations_output + \
+                      gpt2_mlp.activations[1:] + \
+                      post_mlp_add.activations_output
+        self.config = config
+        # Fill Base Model with the generated data
+        super().__init__(activations, layers)
+
+    @staticmethod
+    def from_torch(
+        torch_gpt2_block, x: TensorMoments,
+        config: GPT2ConfigNNTile, next_tag: int):
+        """
+        torch_gpt2_block is HF module for GPT2 Block
+        """
+        layer_norm_input_layer, next_tag = LayerNorm.from_torch(
+            torch_gpt2_block.ln_1,
+            x,
+            next_tag
+        )
+        attention_layer, next_tag = GPT2Attention.from_torch(
+            torch_gpt2_block.attn,
+            layer_norm_input_layer.activations_output[0],
+            layer_norm_input_layer.activations_output[0],
+            layer_norm_input_layer.activations_output[0],
+            config, next_tag)
+        post_attn_add, next_tag = Add.generate_simple(
+            x, attention_layer.activations_output[0],
+            next_tag)
+
+        layer_norm_post_attn_layer, next_tag = LayerNorm.from_torch(
+            torch_gpt2_block.ln_2,
+            post_attn_add.activations_output[0],
+            next_tag
+        )
+        gpt2_mlp_module, next_tag = GPT2MLP.from_torch(
+            torch_gpt2_block.mlp,
+            layer_norm_post_attn_layer.activations_output[0],
+            config, next_tag)
+        post_mlp_add, next_tag = Add.generate_simple(
+            gpt2_mlp_module.activations[-1],
+            post_attn_add.activations_output[0],
+            next_tag)
+
+        nntile_gpt2_decoder = GPT2Block(x, attention_layer,
+                                            gpt2_mlp_module,
+                                            layer_norm_input_layer,
+                                            layer_norm_post_attn_layer,
+                                            post_attn_add,
+                                            post_mlp_add,
+                                            config)
+
+        return nntile_gpt2_decoder, next_tag
+
+    def to_torch(self):
+        config_torch = GPT2ConfigTorch(
+            n_embd=self.config.hidden_size,
+            n_layer=1,
+            n_head=self.config.n_head,
+            n_inner=self.config.intermediate_size,
+            resid_pdrop=0.0,
+            embd_pdrop=0.0,
+            attn_pdrop=0.0,
+            layer_norm_epsilon=self.config.layer_norm_epsilon,
+            scale_attn_weights=True,
+            use_cache=False,
+            add_cross_attention=False,
+        )
+        gpt2_block_torch = GPT2Block_torch(config_torch)
+        gpt2_block_torch.ln_1 = self.layers[0].to_torch()
```

Comment:
add a name to `self.layers[0]` as it is more readable to access elements by name